mlos_core/mlos_core/optimizers/optimizer.py

```diff
@@ -231,7 +231,7 @@ def cleanup(self) -> None:
         Redefine this method in optimizers that require cleanup.
         """
 
-    def _from_1hot(self, config: npt.NDArray) -> pd.DataFrame:
+    def _from_1hot(self, *, config: npt.NDArray) -> pd.DataFrame:
```

Comment:
```suggestion
    def _from_1hot(self, config: npt.NDArray) -> pd.DataFrame:
```

Don't think we need to do this here - it only takes a single argument.

Reply:
I suppose it might be confusing if `context` needs to be 1 hot encoded in the future and we want to reuse this routine, but we can also address that in the future.